index.html

```diff
@@ -444,15 +444,28 @@ <h1>Terminology</h1>
       <dd>A data schema describes the information model and the related payload structure
         and corresponding data items that are passed between <a>Things</a>
         and <a>Consumers</a> during interactions.</dd>
+      <dt>
+        <dfn>Device</dfn>
+      </dt>
+      <dd>A Device is a physical entity that has a network interface.
+      Devices can be described by a <a>Thing Description</a> and are a kind of <a>Thing</a>.
```

Comment:
This is really a "Connected Device", but "Device" is a useful shorthand. I suggest adding both terms to this definition since both terms are used in the body of the text.

Reply:
Arch call on 17.2.: Web Device?
Review usages of connected device?

Reply:
I agree with @benfrancis , let's add Connected Device. We could also include Network Device as a synonym.

Reply:
Just to note that since the terms "Thing" and "Web Thing" have been used as synonyms up until this point [1], giving the two terms separate definitions may create confusion and would require a full review of all specifications to make sure the terms are being used correctly.

Does it really make sense to say that a Thing that isn't a Web Thing can be described by a Web of Things Thing Description?

I won't rehash my arguments for why "MQTT Web Thing" makes no sense ;)

> I suppose you can say "HTTP Web Thing" if necessary (and maybe this is a good name for the Profile we are working on...).

I wouldn't object to that :)

--- 

I have personally often used "Thing" to describe the physical device in the real world represented on the web by a "Web Thing", but that is probably not the common understanding. Those concepts would probably be described as "Device" and "Web Thing" respectively in this latest terminology.

Reply:
Anyways, as I said elsewhere, that is also why I hesitate to put "Web" in front of "Service", since it may imply use of HTTP for some people.  So for now I will also leave that alone.

Reply:
We should review how Web Thing is used.  It may be some cleanup is needed to map (Web Thing, Thing) to (Thing, Device).

Reply:
I added "Connected Device" but did not change the definition of Web Thing.  See discussion below.